src/doc/src/reference/manifest.md

```diff
@@ -125,15 +125,14 @@ set to the latest edition. Setting the `edition` key in
 `[package]` will affect all targets/crates in the package, including test
 suites, benchmarks, binaries, examples, etc.
 
-If the `edition` key is not set to a specific [Rust Edition] in your
-`Cargo.toml`, Cargo will default to 2015.
-
 ```toml
 [package]
 # ...
 edition = '2018'
 ```
 
+If the `edition` key is not set, or the value is not a valid [Rust edition] in your `Cargo.toml`, Cargo will default to 2015.
```

Comment:
I think the "or the value ..." clause isn't quite right here since Cargo doesn't ignore invalid configuration, it'll genereate a parse error in that case. It's only if the key is omitted that we default to 2015

Reply:
Understood I've removed that case entirely.